man/swaync.5.scd

```diff
@@ -366,6 +366,16 @@ config file to be able to detect config errors
 								description: "Command to be executed on click" ++
 					description: A list of actions containing a label and a command ++
 			description: A grid of buttons that execute shell commands ++
+		*volume*++
+			type: object ++
+			css class: widget-volume ++
+			properties: ++
+				label: ++
+					type: string ++
+					optional: true ++
+					default: "Volume" ++
+					description: Text displayed in front of the volume slider ++
+			description: Slider to control pulse volume ++				
```

Comment:
Remove extra tabs on line 378 to fix build error

Reply:
Thx. Already found the error with `libpulse` not being found and the lint issue :) Just looking if I find some more ;)

Reply:
I'll have to rebuild the docker images on my end for the build actions to work. I wonder if GitHub can do this for me? 🤔

Reply:
Not familiar with GitHub on this (only used GitLab CI/CD until now). But I think you should be able to run `docker build` you might have to add docker as a dependency for the action though.

Reply:
Merge from main and it should be fixed :)